pkg/handler/processor/ite6/ite6.go

```diff
@@ -0,0 +1,84 @@
+//
+// Copyright 2022 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ite6
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/guacsec/guac/pkg/handler/processor"
+	"github.com/in-toto/in-toto-golang/in_toto"
+)
+
+type ite6Type string
+
+const (
+	slsaPredicateType ite6Type = "https://slsa.dev/provenance/v0.2"
+)
+
+type ITE6Processor struct {
+}
+
+// ValidateSchema ensures that the document blob can be parsed into a valid data structure
+func (e *ITE6Processor) ValidateSchema(i *processor.Document) error {
+	if i.Type != processor.DocumentITE6Unknown {
+		return fmt.Errorf("expected document type: %v, actual document type: %v", processor.DocumentITE6Unknown, i.Type)
+	}
+
+	_, err := parseStatement(i.Blob)
+
+	return err
+}
+
+// Unpack takes in the document and tries to unpack the provenance.
+// if the predicate is of SLSA type the predicate is stored in the blob
+func (e *ITE6Processor) Unpack(i *processor.Document) ([]*processor.Document, error) {
```

Comment:
I think this just returns `nil, nil`

Reply:
unpack of len 0 output means that it is a leaf node. The guesser would be able to determine if its SLSA or ITE6 so this should not be the case where a processor needs to relabel a document.

Reply:
> The guesser would be able to determine if its SLSA or ITE6 so this should not be the case where a processor needs to relabel a document.
In that case is the ITE6 Processor needed at all? Seems like its not doing anything that the guesser isnt already doing.